werkzeug/formparser.py

```diff
@@ -38,7 +38,7 @@
 def default_stream_factory(total_content_length, filename, content_type,
                            content_length=None):
     """The stream factory that is used per default."""
-    if total_content_length > 1024 * 500:
+    if total_content_length is None or total_content_length > 1024 * 500:
```

Comment:
Beware of this. Since we have no information on content length, the `total_content_length` parameter will be `None`. I decided to default to `TemporaryFile` since we can't possibly know how large the file is going to become...

Reply:
I think it might make sense to use this rollover tempfile that lingers in the stdlib. That starts up to a certain amount in memory and then flushes to fs.

Reply:
Suppose you mean `SpooledTemporaryFile`?

Reply:
Yep, that's the one.

Reply:
👍 Would remove the check here and just always return a spooled temp file.